src/components/cc-tile-status-codes/cc-tile-status-codes.smart.js

```diff
@@ -62,7 +62,16 @@ async function fetchStatusCodes({ apiConfig, signal, ownerId, appId }) {
   const warpToken = await getWarp10AccessLogsToken({ orgaId: ownerId }).then(
     sendToApi({ apiConfig, signal, cacheDelay: ONE_DAY }),
   );
-  return getStatusCodesFromWarp10({ warpToken, ownerId, appId }).then(
-    sendToWarp({ apiConfig, signal, timeout: THIRTY_SECONDS }),
-  );
+  return getStatusCodesFromWarp10({ warpToken, ownerId, appId })
+    .then(sendToWarp({ apiConfig, signal, timeout: THIRTY_SECONDS }))
+    .then(
+      /* @ts-expect-error - FIXME: data is of type StatusCodesBeta but doesn't include `0` */
+      (data) => {
+        // FIXME: This is temporary until we have a proper API and `0` status codes get removed from the backend response
+        if (data[0] != null) {
+          delete data[0];
+        }
+        return data;
+      },
+    );
 }
```

Comment:
**suggestion:** I'm okay with this but I wonder if we could take this opportunity to guard against other mistakes the API could do.

This seems fairly simple to do because we can import `status` from `statuses` and use `status.message` to get the list of status codes we support. Then we can filter out keys that are not part of the status codes we support.
```suggestion
  return getStatusCodesFromWarp10({ warpToken, ownerId, appId })
    .then(sendToWarp({ apiConfig, signal, timeout: THIRTY_SECONDS }))
    .then(
      /** @param {Record<number, number>} data */
      (data) => {
        for (const property in data) {
          if (!(property in status.message)) {
            delete data[property];
          }
        }
        return data;
      },
    );
}
```

This way we could make this a `fix` commit so it's part of the changelog and we can release it on its own or wait for other commits to release several things (if we stick with a chore commit, we cannot release it on its own but maybe we were planning on waiting for other fixes anyway :man_shrugging:)